R/histogramForRaster.R

```diff
@@ -0,0 +1,64 @@
+#' Histogram for Raster Shiny Module
+#'
+#' @description This creates UI for shiny module creating histogram of data contained in raste
+#'
+#' @param id An ID string that corresponds with the ID used to call the module server function
+#' @param title Optional title for the histogram. Any shiny tag can be used.
+#' @param ... Additional parameters passed to \code{\link[shiny]{box}} tag from \pkg{shiny}.
+#' @param plotParameters A list of parameters passed to \code{\link[shiny]{plotOutput}} function from \pkg{shiny} package.
+#'
+#' @importFrom shiny NS plotOutput
+#' @importFrom shinydashboard box
+#' @importFrom shinycssloaders withSpinner
+#'
+#' @rdname histogramForRaster
+#'
+#' @export
+histogramForRasterUI <- function(id, title = "", plotParameters, ...) {
+  ns <- NS(id)
+
+  plotParameters["outputId"] = NULL
+  plotOutputParameters <- c(ns("histogram"), plotParameters)
+
+  box(title,
+      shinycssloaders::withSpinner(
+        do.call(plotOutput, plotOutputParameters)
+        ),
+      ...)
+}
+
+#' Histogram for Raster Shiny Module
+#'
+#' @description
+#'
+#' @param input Shiny server input object
+#' @param output Shiny server output object
+#' @param session Shiny server session object
+#' @param raster Reactive value containing raster
+#' @param scale Number used for scaling heights of histogram bars. When set to 1 (default) histogram bar height represents
+#' amount of raster cells with value from bar interval. If the resolution of raster is known, scale parameter can
+#' be used to transform these heights into the ones representing area covered by cells.
```

Comment:
Please add more details as to what is the unit of scale, for example what does `scale = 2` mean.